.changeset/smart-jeans-tickle.md

```diff
@@ -0,0 +1,24 @@
+---
+"@fuel-ts/abi-coder": patch
+"@fuel-ts/address": patch
+"@fuel-ts/contract": patch
+"@fuel-ts/example-contract": patch
+"forc-bin": patch
```

Comment:
Need to remove forc-bin from here

Reply:
I circumvented this and documented it on the PR description.

Please, have a look and let me know your thoughts about it.